src/index.ts

```diff
@@ -0,0 +1,12 @@
+//
+// This source file is part of the Stanford Biodesign Digital Health Spezi Web Health Components open-source project
+//
+// SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+export * from './components/AllergyForm/AllergyForm'
+export * from './components/PatientForm/PatientForm'
+export * from './components/ObservationForm/ObservationForm'
+export * from './components/PatientsTable/PatientsTable'
```

Comment:
It seems reasonable to export complete modules

```suggestion
export * from './components/AllergyForm'
export * from './components/PatientForm'
export * from './components/ObservationForm'
export * from './components/PatientsTable'
```